### PR TITLE
feat: website implementation

### PR DESCRIPTION
**Pull request by rostyslavvnahornyi**
Hello @kfarr,

The first version of the new website is almost ready!

Buy i have some questions to fix: 
1. Can you give me link to the 3dstreet mastodon.
2. We have form keep me update. How we are go to connect this input with some backend?
3. Can you give me link to the blog page?
4. We need to change the video in overview section(second section)
5. I need to change links to statistic articles in statistics section.
6. Do we have blog posts? How we can use them?
7. Also need info about every link in footer.

And another question: how urgent is it to complete the following tasks: [first](https://github.com/3DStreet/3dstreet-editor/pull/385#issuecomment-1925910271), [second](https://github.com/3DStreet/3dstreet-editor/issues/390)?
Can I start next week?

### CODE REVIEW

**Comment by kfarr** [comment]
Thanks @rostyslavvnahornyi we'll review and provide feedback later today or tomorrow.

No the other items are not urgent and next week is totally ok to start work on them.

Thank you

**Comment by kfarr** [comment]
@rostyslavvnahornyi please note the build fail and address, it looks like some react code meant for front-end browser cannot be parsed by the docusaurus generator

**Comment by rostyslavvnahornyi** [comment]
@kfarr hello!
i fixed build and test deployment is working now, but the rest stages of netlify break. 

```
3:12:38 PM: Failed during stage "building site": Build script returned non-zero exit code: 2
3:12:38 PM: [ERROR] Minimum Node.js version not met :(
3:12:38 PM: [INFO] You are using Node.js v16.20.2, Requirement: Node.js >=18.0.
```

I was trying to disable checking nodejs version in package json but it didnt help me. I think the problem in netlify settings but i havent access to it. I can just see logs there. 
It's very strange because i didnt change any setup for ci/cd.. 
Can you add me to see configuration in netlify or maybe you know how it can be fixed in a couple of clicks?

**Comment by kfarr** [comment]
Thanks Rosty, you're right netlify is using an old npm version 16. I
updated all our netlify builds to 18. I will rerun the build and see if it
works.

**Comment by kfarr** [comment]
@rostyslavvnahornyi updated netlify deploy build node version to 18 fixed it! I also merged in latest changes from main to trigger the rebuild.

Next step is to resolve all of the questions you have shared with us. Now that the netlify build is working it will make it easier to share with others to respond to those.

**Comment by kfarr** [comment]
> Can you give me link to the 3dstreet mastodon.
https://urbanists.social/@3dstreet

> We have form keep me update. How we are go to connect this input with some backend?
mailchimp is the ultimate destination. can you research options around this? can provide api key when needed or generate a web form or whatever.

> Can you give me link to the blog page?
https://www.3dstreet.org/blog/

> We need to change the video in overview section(second section)
To do

> I need to change links to statistic articles in statistics section.


> Do we have blog posts? How we can use them?
> Also need info about every link in footer.



KF notes:
- "3D STREET OVERVIEW" > "3DSTREET OVERVIEW" (remove space between 3D_Street)
- The percentages of the statistics section do not match the design or the percentage figures.

Daria feedback notes:
https://www.figma.com/file/G6tv4Zu0MV66pX7egJs7Zf/3D-Street-landing-page?type=design&node-id=1474%3A535&mode=design&t=nhzrbxU4L00MIhDR-1

**Comment by rostyslavvnahornyi** [comment]
Nice! thank you for the answers, i will get to it as soon as possible

**Comment by rostyslavnahornyi** [comment]
@kfarr Hi! You can see the changes in last commit. There are fixes of bugs that are given above.

About mailchimp: 
I created my free acount and fully tested adding new contacts to audience via url that i got from "Audience => Signup forms => Embedded forms => see image.

This url i placed into env file that now is not hidden. As I understand it, I don’t have access to add an environment variable to the github repository, so I left it in the env files. you should remove it from there when reviewing the code.

In short, you should to create your own account in mailchimp and put your url in env.

image

**Comment by kfarr** [comment]
@rostyslavnahornyi 

attempting to run locally results in:
` Error: Docusaurus could not load module at path "/Users/kieranfarr/dev/3dstreet-docs/docusaurus.config.js"
Cause: Cannot find module 'dotenv'`

**Comment by rostyslavnahornyi** [comment]
Hi @kfarr!

Highest priority issues were fixed, tomorrow i will fix next issues

**Comment by rostyslavnahornyi** [comment]
and I overlooked a little bit that you wanted to try changing the mailchimp env and try testing it, but I also did that and everything works.., which is also good)

**Comment by kfarr** [comment]
@rostyslavnahornyi looks great, merging!

**Comment by rostyslavnahornyi** [comment]
and i fixed lower priority issues but there is one problem with object fit images – it's almost impossible to place all images in blog cards with properly sizes.
 In my opinion, the design of cards should be remade because ~16:9 images to transform to horizontal images look like unreal, there will be strong compression

**Comment by kfarr** [comment]
thanks @rostyslavnahornyi for now we can use css `cover` technique instead of "fill" so that instead of squishing it will maintain the aspect ratio

I will review again today.

Here is the list of 3 remaining issues:

- [ ] mobile sizing of product feature screenshot --> should use "cover" method to preserve aspect ratio while adjusting size
- [ ] mobile sizing of blog post image screenshot --> should use "cover" method to preserve aspect ratio while adjusting size
- [ ] input field for email value is not visible in mobile. See error case:
image

Instead should look like one of these: ([from figma](https://www.figma.com/file/G6tv4Zu0MV66pX7egJs7Zf/3D-Street-landing-page?type=design&node-id=1342-1615&mode=design&t=mA26ibcy4rzEE7N3-4))
image

**Comment by kfarr** [comment]
@rostyslavnahornyi all the issues are resolved locally. there is a github server issue which is preventing deployment https://github.com/orgs/community/discussions/112296#discussioncomment-8766113
